src/state/user/hooks.tsx

```diff
@@ -381,3 +383,37 @@ export function useTrackedTokenPairs(): [Token, Token][] {
     return Object.keys(keyed).map((key) => keyed[key])
   }, [combinedList])
 }
+
+export function useSavedTokens(): Token[] {
```

Comment:
If this is for the favourires, why not call it `favourites` instead of `saved`, remember the user can also add custom tokens. We might get the two things mixed

Reply:
Good point, changed to favourite :+1: